### PR TITLE
[ja] add translation at debug-kit.rst to The DebugKit.safeTld option

### PR DESCRIPTION
**Pull request by nojimage**
Add translation for `DebugKit.safeTld` option.

refs https://book.cakephp.org/3.0/en/debug-kit.html#configuration

### CODE REVIEW

**Comment by okinaka** [comment]
Refs: #5958

**Comment by chinpei215** [comment]
Looks good to me.

File: ja/debug-kit.rst [hunk 1]
```diff
@@ -35,11 +35,17 @@ DebugKit は、CakePHP アプリケーション用のデバッグツールバー
...
+* ``DebugKit.safeTld`` - ローカル開発用のトップレベルドメインのホワイトリストを設定します。
+  これにより安全でないと判断したホスト上でDebugKitを表示できるようになります。 ::
+
+    // 例: http://foo.bar.dev または http://my-shop.local のドメインはローカルとして許可する
+    Configure::write('DebugKit.safeTld', ['dev', 'local', 'example']);
+
+* ``DebugKit.forceEnable`` - DebugKit を強制的に表示します。これを使うときは注意してください。
+  通常は単純にローカルのトップレベルドメインをホワイトリストに追加したほうが安全です。 使用例::
+
+    // Before をロードする前に
```
Comment (okinaka) [inline ja/debug-kit.rst]:
```suggestion
    // DebugKit をロードする前に
```

**Comment by chinpei215** [comment]
@okinaka You are right. I didn't notice that.

**Comment by chinpei215** [comment]
@nojimage Could you please open another pull request to fix the sentence @okinaka pointed out?

**Comment by okinaka** [comment]
@nojimage @chinpei215  I fixed.

**Comment by chinpei215** [comment]
@okinaka Thanks.

**Pull request merged**